Names-values.Rmd

```diff
@@ -562,7 +562,7 @@ Together, these two complications make it hard to predict whether or not a copy
 
 \index{loops!performance}
 \index{for loops|see {loops}}
-Let's explore the subtleties with a case study using for loops. For loops have a reputation for being slow in R, but often that slowness is caused by every iteration of the loop creating a copy. Consider the following code. It subtracts the median from each column of a large data frame: 
+Let's explore the subtleties with a case study using `for` loops. `For` loops have a reputation for being slow in R, but often that slowness is caused by every iteration of the loop creating a copy. Consider the following code. It subtracts the median from each column of a large data frame: 
```

Comment:
I don't think these should be given a code font because I'm using the term "for loop" generally.

Reply:
Done!